runtime/common/src/claims.rs

```diff
@@ -607,27 +607,30 @@ mod tests {
 	// or public keys. `u64` is used as the `AccountId` and no `Signature`s are required.
 	use sp_runtime::{traits::{BlakeTwo256, IdentityLookup, Identity}, testing::Header};
 	use frame_support::{
-		impl_outer_origin, impl_outer_dispatch, assert_ok, assert_err, assert_noop, parameter_types,
+		assert_ok, assert_err, assert_noop, parameter_types,
 		ord_parameter_types, weights::{Pays, GetDispatchInfo}, traits::ExistenceRequirement,
 		dispatch::DispatchError::BadOrigin,
 	};
 	use pallet_balances;
-	use super::Call as ClaimsCall;
-
-	impl_outer_origin! {
-		pub enum Origin for Test {}
-	}
-
-	impl_outer_dispatch! {
-		pub enum Call for Test where origin: Origin {
-			claims::Claims,
+	use crate::claims;
+	use claims::Call as ClaimsCall;
+
+	type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+	type Block = frame_system::mocking::MockBlock<Test>;
+
+	frame_support::construct_runtime!(
+		pub enum Test where
+			Block = Block,
+			NodeBlock = Block,
+			UncheckedExtrinsic = UncheckedExtrinsic,
+		{
+			System: frame_system::{Module, Call, Config, Storage, Event<T>},
+			Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>},
+			Vesting: pallet_vesting::{Module, Call, Storage, Config<T>, Event<T>},
+			Claims: claims::{Module, Call, Storage, Config<T>, Event<T>},
```

Comment:
```suggestion
			Claims: claims::{Module, Call, Storage, Config<T>, Event<T>, ValidateUnsigned},
```
Maybe we can make test declare it, even though tests make no use of it through runtime